md/getting_started.md

```diff
@@ -1,56 +1,46 @@
 # Getting started
 
-Bonita Intelligent Continuous Improvement installation and configuration instruction. 
+Bonita Intelligent Continuous Improvement Add-on installation and configuration instruction. 
 
-This chapter cover *Quick start* and *Production* installation.
+This chapter covers *Evaluation* and *Production*, two different installation modes.
 
 ## Download
 
-Download ICI from [Bonitasoft Customer Portal](https://customer.bonitasoft.com/).
+Download ICI Add-on from [Bonitasoft Customer Portal](https://customer.bonitasoft.com/).
 
 The archive contains:
-* The installation guide: `installation-guide.md`
+* An installation guide: `installation-guide.md`
 * The ICI application: `ici-application-<version>.zip`
-* The Living applications
+* Two Living Applications: Case Monitoring with predictions, and Configuration of the first application
 * An installer: `bin/bonita-ici`
 
 ## Installation
 
-Installation can be done in a *Quick start* or *Production* mode. 
- 
-* *Quick start* mode is designed to try out the ICI module on an existing Bonita platform.
-* *Production* mode should be used when using ICI module for production.
-
-## Pre-requisites
-
-* A Bonita platform
-* Java
-* Docker (for *Quick start*)
+Installation can be done in either *Evaluation* or *Production* mode. 
 
 :::info
 Check all [pre-requisites](./prerequisites.md) prior to install.
 :::
 
-If Bonita engine database vendor is oracle, get JDBC driver jar from Oracle (not included in distribution).
+* *Evaluation mode* mode is designed to try out the ICI add-on on an existing Bonita platform.
+* *Production* mode should be used when using ICI add-on for production.
```

Comment:
I think we need to also say that the big difference is that in Evaluation, we provide Elasticsearch where in Production, Elasctisearch has to already exist in our customer's environment.

Reply:
i'm adding this in installation guide in a separate PR, but we can explain difference here also

Reply:
added difference just above